product-service/openapi.yaml

```diff
@@ -0,0 +1,109 @@
+openapi: 3.0.0
+info:
+  title: Products API
+  version: 1.0.0
+  description: API for managing products
+servers:
+  - url: https://qyflqg7cf4.execute-api.eu-west-1.amazonaws.com/prod
```

Comment:
please add a second OpenAPI file for the `GET /import` API.

Reply:
Ok, thank you for reminder